README.md

```diff
@@ -1,55 +1,85 @@
-# cloudflare-jwt-verifier
-Cloudflare worker that verifies incoming requests have a valid JSON Web Token (JWT) before the request is forwarded on to your infrastructure
+# Approov Quick Start - Cloudflare Worker
+
+Approov Cloudflare worker that verifies incoming requests have a valid [Approov](https://approov.io) JSON Web Token (JWT) before the request is forwarded on to your infrastructure or to a third party service.
+
+Approov is meant to be used when you want to ensure that only your mobile app can talk to your mobile API, and you can learn in more detail how this is done [here](https://approov.io/approov-in-detail.html).
```

Comment:
This may not be a wise link given the web site change - do we use it elsewhere?

Reply:
This is used a lot everywhere, thus the new website will need to have a redirect to it.

Reply:
Okay fine - but make sure Skip knows...